### PR TITLE
Hibernate OGM 5.4.0.Alpha1

### PR DESCRIPTION
**Pull request by fax4ever**

### CODE REVIEW

File: ogm/releases/5.4/index.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * Neo4J to 3.3.5
+
+==== Module Distribution
+
+ * WildFly modules as feature pack distribution is now available
+
+==== Infinispan
+
+ * Caches created by the dialect are now transactional as default
+ * Caches created by the dialect could be configured using a cache configuration
```
Comment (DavideD) [inline ogm/releases/5.4/index.adoc]: s/could be/can be/

File: ogm/releases/5.4/index.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * Hibernate ORM to 5.3.0.CR1
+ * Hibernate Search to 5.10.0.Beta2
+ * WildFly to 12.0.0
+ * Infinispan to 9.2.0.Final
+ * MongoDB to 3.6.3
+ * Neo4J to 3.3.5
+
+==== Module Distribution
+
+ * WildFly modules as feature pack distribution is now available
```
Comment (DavideD) [inline ogm/releases/5.4/index.adoc]: WildFly modules are now available as feature packs

File: _data/projects/ogm/releases/5.4/5.4.0.Alpha1.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+version: 5.4.0.Alpha1
+version_family: 5.4
+date: 2018-04-16
+stable: false
+announcement_url: http://in.relation.to/2018/04/16/hibernate-ogm-5-4-Alpha1-released/
+summary: Wildfly 12 Feature Pack - JPA 2.2 - Hibernate 5.3 - Infinispan 9.2
```
Comment (gsmet) [inline _data/projects/ogm/releases/5.4/5.4.0.Alpha1.yml]: s/Wildfly/WildFly/
s/Hibernate/Hibernate ORM/

File: _data/projects/ogm/releases/5.4/series.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+summary: Wildfly 12 Feature Pack - JPA 2.2 - Hibernate 5.3 - Infinispan 9.2
```
Comment (gsmet) [inline _data/projects/ogm/releases/5.4/series.yml]: s/Wildfly/WildFly/
s/Hibernate/Hibernate ORM/

File: ogm/releases/5.4/index.adoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+==== Infinispan
+
+ * Caches created by the dialect are now transactional as default
```
Comment (gsmet) [inline ogm/releases/5.4/index.adoc]: I would say "by default".

File: ogm/releases/5.4/index.adoc [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+ * Caches created by the dialect could be configured using a cache configuration
+
+==== MongoDB
+
+ * Stored procedure with positional parameters in now supported
+ * Native query enhancement
```
Comment (DavideD) [inline ogm/releases/5.4/index.adoc]: Maybe change this to: New MongoDB CLI operations supported as native queries

File: ogm/releases/5.4/index.adoc [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+ * WildFly modules as feature pack distribution is now available
+
+==== Infinispan
+
+ * Caches created by the dialect are now transactional as default
+ * Caches created by the dialect could be configured using a cache configuration
+
+==== MongoDB
+
+ * Stored procedure with positional parameters in now supported
```
Comment (gsmet) [inline ogm/releases/5.4/index.adoc]: Stored procedures with positional parameters are now supported

File: ogm/releases/5.4/index.adoc [hunk 8]
```diff
@@ -0,0 +1,35 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: ogm
+:awestruct-series_version: "5.4"
+
+=== Core
+
+Hibernate OGM is now compatible with Hibernate ORM 5.3 and JPA 2.2!
+
+==== Components upgrade
```
Comment (gsmet) [inline ogm/releases/5.4/index.adoc]: I would write it "Component upgrades"

**Comment by DavideD** [comment]
Looks good to me, feel free to merge after the changes

File: ogm/releases/5.4/index.adoc [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+ * Caches created by the dialect could be configured using a cache configuration
+
+==== MongoDB
+
+ * Stored procedure with positional parameters in now supported
+ * Native query enhancement
+
+==== Fixes
+
+ * Bug fixes related to projections and native query pagination
```
Comment (gsmet) [inline ogm/releases/5.4/index.adoc]: Aren't they already in the last 5.3? As we just started the 5.4 release cycle, I don't think we need to put them here.

Comment (fax4ever) [reply ogm/releases/5.4/index.adoc]: ok

**Comment by fax4ever** [comment]
Thank you @gsmet and @DavideD.

**Pull request merged**